js/dotprompt/README.md

```diff
@@ -1,7 +1,7 @@
 # Genkit
 
-The sources for this package are in the main [Genkit](https://github.com/firebase/genkit) repo. Please file issues and pull requests against that repo.
+The sources for this package are in the main [Genkit](https://github.com/google/genkit) repo. Please file issues and pull requests against that repo.
 
-Usage information and reference details can be found in [Genkit documentation](https://firebase.google.com/docs/genkit).
+Usage information and reference details can be found in [Genkit documentation](https://developer.google.com/genkit).
 
```

Comment:
Did you mean to change this file? I think the firebase links are the correct ones. 😄

Reply:
Was copied over by mistake. Fixed.